src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -28,7 +28,146 @@ namespace CoordinateMaps {
  *  0) and spherical (a sphericity of 1).
  *
  *  The first two logical coordinates correspond to the two angular coordinates,
- *  and the third to the radial coordinate
+ *  and the third to the radial coordinate.
+ *
+ *  The Wedge3D map is constructed by linearly interpolating between a bulged
+ *  face of radius `radius_of_other_surface` to a spherical face of
+ *  radius `radius_of_spherical_surface`, where the radius of the bulged face
+ *  is defined to be the radius of the sphere circumscribing the bulge.
+ *
+ *  We make a choice here as to whether we wish to use the logical coordinates
+ *  parameterizing these surface as they are, in which case we have the
+ *  equidistant choice of variables, or whether to apply a tangent map to them
+ *  which leads us to the equiangular choice of variables. The equiangular
+ *  variables are:
```

Comment:
What is "variables"? I interpret these maps as producing an equidistant/equiangular grid and coordinates, so I would have used one of {grid, coordinates} here. Are you using "variables" to refer to the map itself, or are you making a distinction between equidistant vs. equiangular evolved variables?

Reply:
I agree that "coordinates" should be used here :+1: